README.md

```diff
@@ -106,3 +210,4 @@ using [starrocks](./helm-charts/charts/kube-starrocks/charts/starrocks) Helm Cha
 
 - In [doc](./doc) directory, you can find more documents about how to use StarRocks Operator.
 - In [examples](./examples/starrocks) directory, you can find more examples about how to write StarRocksCluster CR.
+- [Documentation on docs.starrocks.io](https://docs.starrocks.io/docs/deployment/sr_operator/#upgrade-starrocks-cluster)
```

Comment:
need to change the URL to https://docs.starrocks.io/docs/deployment/sr_operator/

Reply:
fixed